src/network/internal_types.rs

```diff
@@ -26,6 +26,8 @@ pub struct SetupNetwork {
     pub isolation: IsolateOption,
     /// port used for the dns server
     pub dns_port: u16,
+    /// outbound address for SNAT
```

Comment:
you should define
```
#[serde(skip_serializing_if = "Option::is_none")]
#[serde(default)]
```
for this field then the json parsing should work from the previous versions